### ISSUE TITLE
Issue with RxJava 1.1.8

### ISSUE THREAD

**Issue by ebonet-zz** [ubiratansoares/rxassertions#6]
Lib RxJava dependency (1.1.7) conflicts with latest 1.1.8. A quick fix is to use this code to declare the dependency on build.gradle : 

```
androidTestCompile('com.github.ubiratansoares:rxassertions:v0.3.1') {
   exclude group: 'io.reactivex', module: 'rxjava'
}
```

**Comment by vanniktech** [comment]
#7 will fix this

**Comment by ebonet-zz** [comment]
I believe #7 is just a temporary fix. Soon a new version of RxJava will be released and the version will again be outdated.

**Comment by vanniktech** [comment]
That is true yeah

**Comment by ubiratansoares** [comment]
Sorry by late reply guys. I will figure out how to get right on packaging/releasing mechanism, to avoid these classpath conflicts soon (when I get some time). By now I`ll upgrade to 1.1.9 at next release

**Comment by ubiratansoares** [comment]
Just released 0.3.2
